src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -320,7 +321,8 @@ public function getAttributes()
     {
         return [
             'data-editor' => 'tinyMCE', // Register ss.editorWrappers.tinyMCE
-            'data-config' => Convert::array2json($this->getConfig())
+            'data-config' => Convert::array2json($this->getConfig()),
+            'style' => 'height:350px'
```

Comment:
Hey, thanks for looking into this. The min_height works on first loading the page.
For some reason, when you save the document, TinyMCE re-renders and does not follow original height. Thus adding 'style' here works for me during that time.